src/js/login/containers/Main.jsx

```diff
@@ -69,13 +52,29 @@ class Main extends React.Component {
     }).then(response => {
       return response.json();
     }).then(json => {
-      // console.log(json);
       const myLogoutUrl = json.logout_via_get;
       const receiver = window.open(myLogoutUrl, '_blank', 'resizable=yes,top=50,left=500,width=500,height=750');
       receiver.focus();
     });
   }
 
+  checkTokenStatus() {
+    if (localStorage.userToken) {
+      if (moment() > moment(localStorage.entryTime).add(45, 'm')) {
+        if (confirm('Your login time has expired. Would you like to login again?')) {
```

Comment:
@AnitraAppa -- The user will get logged out after a certain amount of time even if they are actively doing things.